x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/details/index.tsx

```diff
@@ -408,6 +415,28 @@ const RuleDetailsPageComponent = () => {
     }
   }, [rule]);
 
+  const onSkipFocusBeforeEventsTable = useCallback(() => {
```

Comment:
I think that I will create a specific component for it because I found the same pattern three times.

`x-pack/plugins/security_solution/public/network/pages/network.tsx`
`x-pack/plugins/security_solution/public/hosts/pages/hosts.tsx`